datawig/imputer.py

```diff
@@ -446,24 +465,27 @@ def __filter_predictions(predictions: dict,
 
     def __predict_above_precision_mxnet_iter(self,
                                              mxnet_iter: ImputerIterDf,
-                                             precision_threshold: float = 0.95) -> dict:
+                                             precision_threshold: float = 0.95,
+                                             filtering_mode: str = 'min') -> dict:
         """
         Imputes values only if predictions are above certain precision threshold,
             determined on test set during fit
 
         :param mxnet_iter: iterator, see ImputerIter in iterators.py
         :param precision_threshold: don't predict if predicted class probability is below
             this precision threshold
+        :param: filtering_mode: whether to filter by expected aggregate precision or by minimum per item precision
```

Comment:
specify available options in comment.

is this something that end-users will be able to specify? can you explain when to use one versus the other?

Reply:
I would propose to remove the old method. It's not visible for the user at the moment.

Reply:
Fine with me.